### DISCUSSION TITLE
Differential Cell2Cell signalling analysis

### DISCUSSION THREAD

**Discussion by varunkas** [msraredon/NICHES#26] [Q&A]
Hi,

I am working on a single-cell dataset that has two experimental conditions. I want to compare the differential Cell2Cell signalling between these conditions.

However, these two conditions have very different number of cells and different sequencing depth (i.e., different median nCount_RNA and nFeature_RNA). 

I can see that downsampling in order to equalize the UMI counts and cell numbers between these two conditions makes a difference to the differential CellToCell signalling. But I am not sure what is closer to the ground truth? Is having equal sequencing depth and number of cells necessary for differential Cell2Cell signalling? Thanks!

**Comment by RaredonLab** [comment]
This is a good question. I think the right answer is that yes, having closer information capture depth will make things more accurate. Having equal cell numbers is I think less important for this specific test, because the CellToCell metric is fairly agnostic to cell type representation. Does this answer your question?

**Reply by varunkas** [reply]
Thanks so much. Yes, partly. So, would you recommend that I downsample the UMI-counts to even them out a bit? Thanks!
